couchbase-persistence/couchbase-persistence-scala-sbt/build.sbt

```diff
@@ -14,7 +14,7 @@ val scalaTest = "org.scalatest" %% "scalatest" % "3.0.8" % Test
 lazy val `couchbase-persistence-scala-sbt` = (project in file("."))
   .aggregate(`hello-api`, `hello-impl`)
 
-val akkaVersion = "2.6.0-RC2"
+val akkaVersion = "2.6.1"
```

Comment:
Remove this val and use `com.lightbend.lagom.core.LagomVersion.akka` instead.

Reply:
```suggestion
val akkaVersion = com.lightbend.lagom.core.LagomVersion.akka
```

Reply:
I'm just trying out if it's possible to make a cod esuggestion on a Pr that's been merged already.